big-practice/src/js/helpers/showing-modal.helper.js

```diff
@@ -0,0 +1,10 @@
+export class ShowingModal {
+  /**
+   * Show modal
+   *
+   * @param {element} element - Element of modal
```

Comment:
please double check type of `element` param.

Reply:
Updated!